supported-devices/devices.json

```diff
@@ -1812,7 +1812,7 @@
         "default_wpmethod": "<a href=\"https://wiki.mrchromebox.tech/Firmware_Write_Protect#Hardware_Write_Protection\" title=\"Firmware Write Protect\">CR50</a>, jumper",
         "default_rwLegacy": true,
         "default_fullrom": true,
-        "default_windows": "Not supported.",
+        "default_windows": "Waiting on drivers to be signed.",
```

Comment:
AFAIK, CZN is not actively being worked on, not even Linux support.

Reply:
Drivers exist for Nipperkin and Dewatt on Coolstar's Windows Install Helper site and there is a banner that says "the audio driver is waiting to be signed by Microsoft", could you please confirm no Windows support?

Reply:
In lieu of CZN Linux support not being worked on currently, is there info you would like me to add to Linux Notes?

Reply:
AFAIK, linux only has issues with the speaker.

Reply:
I've decided to revert these changes after taking into account issues with RW_Legacy on CZN as well. You can revisit this in the future when speakers are working on Linux, Windows support is confirmed and/or Google fixes their firmware.

Reply:
My mistake, CZN is supported on windows. I'll go fix that.